volatility3/framework/plugins/windows/iat.py

```diff
@@ -69,11 +69,23 @@ def _generator(self, procs):
                     layer_name=proc_layer_name,
                 )
 
-                for offset, data in dos_header.reconstruct():
-                    pe_data.seek(offset)
-                    pe_data.write(data)
+                try:
+                    for offset, data in dos_header.reconstruct():
+                        pe_data.seek(offset)
+                        pe_data.write(data)
+                except (exceptions.InvalidAddressException, ValueError) as excp:
+                    vollog.debug(
```

Comment:
Should this not be more visible (although possibly shorter), if we know something bad has happened?

Reply:
I made it warning now.